Jenkinsfile

```diff
@@ -58,25 +61,25 @@ pipeline {
       }
     }
 
-    stage("Run tests") {
-      when {
-        anyOf {
-          branch "main"
-          branch "develop"
-          branch "bugfix/*"
-          branch "hotfix/*"
-          branch "release/*"
-          branch "tickets/*"
-          branch "PR-*"
-        }
-      }
-      steps {
-        script {
-          sh "docker build -f docker/Dockerfile-test -t love-frontend-test  ."
-          sh "docker run love-frontend-test"
-        }
-      }
-    }
+    // stage("Run tests") {
```

Comment:
In general we prefer not to have commented out lines in our code and there suddenly are many in this file. Are these absolutely necessary? If not, please remove them.

Reply:
Also, are you sure you want to disable running the unit tests?

Reply:
Sorry, this was intended for testing a further step. I'll make the change. Thanks!